cypress/integration/ete-okta/sign_out.2.cy.ts

```diff
@@ -35,11 +35,15 @@ describe('Sign out flow', () => {
         // attempt to sign out and redirect to reset password as it's on same domain
```

Comment:
I take it there was no relevance to being redirected to `reset-password`, apart from it being on the same domain?

Reply:
updated the comment